submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,382 @@
+
+- **Speed modules:** One effect of speed modules is increasing power usage ("P") just like efficiency modules, but in the opposite direction, hence reducing our energy gains. The other effect of speed modules is increasing the crafting speed of machines ("C"), which as seen from the equation, reduces the power usage because the added speed makes the machine work for a shorter time for each crafting cycle. For every speed module, the increase in P is greater than the increase in C, which means that the overall effect of any speed module is an increase in E, which results in decreases in energy gains overall. However, speed modules can be used alongside productivity modules to mitigate the slowdown they cause, and this might provide energy gains overall.
+
+- **Productivity modules:** They have the most interesting effects. They increase power usage ("P") and decrease crafting speed ("C"), both of which lead to an increase in E, which decreases the overall energy gains. On the other hand, productivity modules increase the amount of processed fuel that is produced. Furthermore, this increase is compounded on every step of processing, which leads to exponential increases in the fuel output overall! The energy gains from the extra fuel might outweigh all the extra power consumption.
+
+{% include image.html
+
+src= 'src'
+ alt='Module effects on power consumption'
+ caption='Module effects on power consumption'
+
+%}
+
+Now let's demonstrate the use of modules. First, let's look at the case of efficiency modules in coal liquification with heavy oil cracking. Refineries and chemical plants have three module slots, so we can place three efficiency 1 modules to reduce power consumptions by 80% for each machine:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing coal into solid fuel with heavy oil cracking, at minimum power consumption'
+ caption='Processing coal into solid fuel with heavy oil cracking, at minimum power consumption'
+
+%}
+
+Now we recalculate the energy gain with the decreased power consumption:
+
+`Energy gain = 88.5 - 40 - 1.5 - 0.42 - 0.1365 - 0.5775 - 0.042 = 45.824 MJ`
+
+`Percent gain = 45.824 MJ / 40 MJ = 114.56% ≈ 115%` 
+
+The energy gain was 103% with no modules. We see that efficiency modules increase the energy gain from 103% to 115% This increase is nice, but it is not that significant, since the processing energy cost was already relatively low. Nevertheless, efficiency 1 modules being cheap to produce makes this option worthwhile. 
+
+Now, let us look at the same example with three productivity 3 modules in every machine instead of efficiency 1 modules. Despite increasing the power consumption they also increase the amount of fuel produced, by 10% per machine per module:
```

Comment:
```suggestion
Now, let us look at the same example with three productivity 3 modules in every machine, instead of efficiency 1 modules. Despite increasing the power consumption, each module also increases the amount of fuel produced by 10% per machine:
```